static/js/actions/financial_aid.js

```diff
@@ -40,3 +44,28 @@ export const addFinancialAid = (income: number, currency: string, programId: num
       });
   };
 };
+
+export const REQUEST_SKIP_FINANCIAL_AID = 'REQUEST_SKIP_FINANCIAL_AID';
+export const requestSkipFinancialAid = createAction(REQUEST_SKIP_FINANCIAL_AID);
+
+export const RECEIVE_SKIP_FINANCIAL_AID_SUCCESS = 'RECEIVE_SKIP_FINANCIAL_AID_SUCCESS';
+export const receiveSkipFinancialAidSuccess = createAction(RECEIVE_SKIP_FINANCIAL_AID_SUCCESS);
+
+export const RECEIVE_SKIP_FINANCIAL_AID_FAILURE = 'RECEIVE_SKIP_FINANCIAL_AID_FAILURE';
+export const receiveSkipFinancialAidFailure = createAction(RECEIVE_SKIP_FINANCIAL_AID_FAILURE);
+
+export const skipFinancialAid = (programId: number): Dispatcher<*> => {
+  return (dispatch: Dispatch) => {
+    dispatch(requestSkipFinancialAid());
+    return api.skipFinancialAid(programId).then(
+      () => {
+        dispatch(receiveSkipFinancialAidSuccess());
+        dispatch(fetchCoursePrices());
+        dispatch(fetchDashboard());
+        return Promise.resolve();
```

Comment:
This promise will resolve before the course prices or dashboard fetch is completed. I'm not sure we need that dependency though